src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Tree/Dependencies/TreeView/GroupedByTargetTreeViewProviderTests.cs

```diff
@@ -32,7 +33,7 @@ public void GrouppedByTargetTreeViewProvider_WhenEmptySnapshot_ShouldJustUpdateD
 
             // Act
             var provider = new GroupedByTargetTreeViewProvider(treeServices, treeViewModelFactory, commonServices);
-            var resultTree = provider.BuildTree(dependenciesRoot, snapshot);
+            var resultTree = await provider.BuildTreeAsync(dependenciesRoot, snapshot);
```

Comment:
ConfigureAwaits for these? Once we reenable analyzers it'll complain about these.